platform/ethereum/compound/compound.go

```diff
@@ -0,0 +1,216 @@
+package compound
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/trustwallet/blockatlas/pkg/blockatlas"
+)
+
+// Compound Lending Provider
+
+type Provider struct {
+	client Client
+}
+
+func Init(api string) *Provider {
+	return &Provider{
+		client: Client{blockatlas.InitClient(api)},
+	}
+}
+
+const (
+	_providerName                   = "compound"
```

Comment:
what is the purpose of `_` which code style language is this?

Reply:
Read it in some of the go recommendations recently; globals should be more descriptive names (as opposed to local vars), and prefixed by underscore; i can't find ref right now.  I've removed the underscores.